tests/foreman/cli/test_import.py

```diff
@@ -1210,6 +1210,8 @@ def test_negative_reimport_cv(self):
                     'csv-file': files['custom-channels'],
                     'dir': os.path.join(tmp_dir, 'exports/CHANNELS'),
                 })
+                if bz_bug_is_open('1263650'):
```

Comment:
This BZ is CLOSED as WONTFIX resolution, PM says there is no plan to fix this soon,
so maybe it is better to skip the if and call wait_for_publish permanently?